ts/src/bingx.ts

```diff
@@ -1770,6 +1793,98 @@ export default class bingx extends Exchange {
         return this.parseOrder (order, market);
     }
 
+    async createOrders (orders: OrderRequest[], params = {}) {
```

Comment:
did you test what happens when one of the orders fails?

Reply:
I think for spot the order status will be failed and for swap it won't show the id but I can double check that later today

Reply:
For spot if one of the orders isn't viable, none of the orders go through and I get this error:
```
[BadRequest] bingx {"code":100400,"msg":"The current system is busy, please try again later","debugMsg":""}
```

For swap it seems to depend on the error:
If the amount is too small for one of the orders, neither of the orders go through and it returns this response:
```
[InvalidOrder] bingx amount of BTC/USDT:USDT must be greater than minimum amount precision of 4
```
If one of the orders has a different type of invalid parameter, one of the orders goes through the other doesn't and an error message is returned with the information of the passed order in the error message:
```
[BadRequest] bingx {"code":80001,"msg":"Insufficient margin","data":{"orders":[{"symbol":"BTC-USDT","orderId":1720972673561088000,"side":"BUY","positionSide":"LONG","type":"LIMIT","clientOrderID":"","workingType":""}]}}
```
```
[BadRequest] bingx {"code":80001,"msg":"Order price should be higher than 31515.7","data":{"orders":[{"symbol":"BTC-USDT","orderId":1720974281279754240,"side":"BUY","positionSide":"LONG","type":"LIMIT","clientOrderID":"","workingType":""}]}}
```